nab/detectors/bayes_changept/bayes_changept_detector.py

```diff
@@ -0,0 +1,164 @@
+"""
+Anomaly detection algorithm that implements Bayesian changepoint detection, as
```

Comment:
Good comments. Somewhat redundant with README - suggest we merge both into this file.